modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,45 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.3--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(amp_input)
+    path(faa_folder)
+    val(outdir)
+
+    output:
+    tuple val(meta), path("${outdir}")          , emit: results_dir
+    path("${outdir}/*/*_diamond_matches.txt")   , emit: txt
+    path("${outdir}/*/*_ampcombi.csv")          , emit: csv
+    path("${outdir}/*/*_amp.faa")               , emit: faa
+    path("${outdir}/*.log")                     , optional:true, emit: log
```

Comment:
Why is the log optional? If anything one would expect this to be the only 'stable' output from  a tool (even if it can't _do_ anything with the input.

Reply:
It is optional in the tool and would have to be activated, therefore optional. But we can activate it and make it mandatory in the module.

Reply:
If you have to do a bug fix, I would suggest making it default
 Logs are _always_ useful (for provenance etc), so it would make more sense to make it opt-out of you want to have it optional still (--ignore-log or something)

Reply:
It prints it to the console otherwise, so you can see the progress of the tool

Reply:
But we can't see that in a NXF pipeline, so I would suggest hardcoding it :)